modules/swagger-codegen/src/main/resources/Javascript/client.mustache

```diff
@@ -86,9 +86,18 @@ function getRequestOptions(path, method, headers, body, session) {
     body: body,
   };
 
-  if (session && session.cookies)
+  if (session && session.cookies) {
```

Comment:
All of this code needs to come before the `var options = {}` assignment.

Reply:
@shcallaway how about assigning xsrf to option.headers?

moving the block above would lose scope of assigning cookieJar to options.